### PR TITLE
Hackweek Firebase Performance -> Crashlytics Interop

### PR DESCRIPTION
**Pull request by samedson**
Hackweek

### CODE REVIEW

File: Interop/Crashlytics/Public/FIRCrashlyticsInterop.h [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * Adds logging that is sent with your crash data. The logging does not appear  in the
+ * system.log and is only visible in the Crashlytics dashboard.
+ *
+ * @param msg Message to log
+ */
+- (void)log:(NSString *)msg;
```
Comment (pydichandra) [inline Interop/Crashlytics/Public/FIRCrashlyticsInterop.h:29]: Is this same as breadcrumb in dashboard?
If not, do we need to define a method to log breadcrumbs as well here?

File: FirebasePerformance.podspec [hunk 2]
```diff
@@ -32,6 +32,7 @@ Firebase Performance library to measure performance of Mobile and Web Apps.
     'FirebaseCore/Sources/Private/*.h',
     'FirebaseInstallations/Source/Library/Private/*.h',
     'FirebaseRemoteConfig/Sources/Private/*.h',
+    'Interop/Crashlytics/Public/*.h',
```
Comment (paulb777) [inline FirebasePerformance.podspec:35]: Fine for hackweek, but the Interop directory is deprecated and new ones should be created like FirebaseAppCheck/Sources/Interop

Comment (samedson) [reply FirebasePerformance.podspec:35]: Ooh cool thanks for that tip

File: Interop/Crashlytics/Public/FIRCrashlyticsInterop.h (hunk 1 above)
Comment (samedson) [reply Interop/Crashlytics/Public/FIRCrashlyticsInterop.h:29]: Yep that'll do breadcrumbs

File: FirebasePerformance/Sources/AppActivity/FPRSessionManager.m [hunk 3]
```diff
@@ -92,6 +94,23 @@ - (void)updateSessionId:(NSNotification *)notification {
...
+  NSError *error;
+  NSData *crashlyticsSessionJsonBreadcrumb =
+      [NSJSONSerialization dataWithJSONObject:crashlyticsSessionBreadcrumb options:0 error:&error];
+  if (!crashlyticsSessionJsonBreadcrumb) {
+    NSLog(@"Got an error: %@", error);
+  } else {
+    NSString *jsonString = [[NSString alloc] initWithData:crashlyticsSessionJsonBreadcrumb
+                                                 encoding:NSUTF8StringEncoding];
+    // Getting the FirePerf shared instance here. Is there a better way to do that internally?
+    [[FIRPerformance sharedInstance].crashlytics log:jsonString];
```
Comment (samedson) [inline FirebasePerformance/Sources/AppActivity/FPRSessionManager.m:112]: Only weirdish part, but might be ok

**Pull request closed without merging**